control/xferfcn.py

```diff
@@ -225,6 +225,7 @@ def __init__(self, *args, **kwargs):
                     break
             if not static:
                 break
+        self._static = static
```

Comment:
In the algorithm aboveI  think  the cryptic [refs_ok] argument may deserve a comment. Is that option so that we can accommodate non numpy entries, or because each entry in the array is itself a numpy array?

Reply:
`[refs_ok]` is required because each entry in the array is itself a numpy array.  I'll add a comment to that effect.